doc/api/crypto.md

```diff
@@ -3533,22 +3533,31 @@ the corresponding digest algorithm. This does not work for all signature
 algorithms, such as `'ecdsa-with-SHA256'`, so it is best to always use digest
 algorithm names.
 
-### `crypto.diffieHellman(options)`
+### `crypto.diffieHellman(options[, callback])`
 
 <!-- YAML
 added:
  - v13.9.0
  - v12.17.0
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/57274
+    description: Optional callback argument added.
 -->
 
 * `options`: {Object}
   * `privateKey`: {KeyObject}
   * `publicKey`: {KeyObject}
-* Returns: {Buffer}
+* `callback` {Function}
+  * `err` {Error}
+  * `secret` {Buffer}
+* Returns: {Buffer} if the `callback` function is not provided.
```

Comment:
```suggestion
* Returns: {Buffer} if the `callback` function is not provided and `undefined` if it is.
```

Reply:
None of the existing callbacks document optional callback interaction with return this way. I'll leave this in line with the existing two methods (sign and randomBytes), rather than introducing inconsistency in docs.

Reply:
Ok, that's fine.